content/docs/guides/crossguard/faq.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Frequently Asked Questions
+linktitle: Frequently Asked Questions
+
+menu:
+  userguides:
+    parent: crossguard
+    identifier: crossguard-faq
+---
+{{% crossguard-preview %}}
+
+1. *Is CrossGuard open source?*
```

Comment:
It might be worth using headings fro each question, like we do here: https://www.pulumi.com/docs/troubleshooting/faq/

Then the list of questions will show up on the right-hand nav bar.